### DISCUSSION TITLE
Any consent decision rejects all categories with an acceptType of 'custom'

### DISCUSSION THREAD

**Discussion by codewithzac** [orestbida/cookieconsent#737] [Q&A] [answered]
Hi there,

Working on an old Magento site and am running into a situation where any consent decision (doesn't matter whether it's all / necessary / custom) results in a consent state with acceptType = 'custom' and all categories showing up in the rejectedCategories array.

I know the config is OK, because trying the same config on another site works as expected (clicking reject in the prompt gives a consent state with acceptType = necessary, etc).

Unfortunately the URL for the site isn't public, otherwise I'd share. Does anyone have any recommendations?

**Comment by orestbida** [comment, marked as answer]
What's the Magento version?

Perhaps its related to this https://github.com/orestbida/cookieconsent/issues/727.

**Comment by codewithzac** [comment]
Legend! Same Magento version, same fix needed. Thank you very much!! 😁
